src/web/WebSchedulerService.cpp

```diff
@@ -245,7 +245,7 @@ void WebSchedulerService::publish(const bool force) {
     JsonDocument doc;
     bool         ha_created = ha_registered_;
     for (const ScheduleItem & scheduleItem : *scheduleItems_) {
-        if (!scheduleItem.name.empty() && !doc.containsKey(scheduleItem.name)) {
+        if (!scheduleItem.name.empty() && !doc[scheduleItem.name].is<const char *>()) {
```

Comment:
depending on bool format this can be bool, int, const char*, better use JsonVariantConst